src/server/pps/server/api_server.go

```diff
@@ -1130,6 +1140,17 @@ func (a *apiServer) scaleUpWorkers(ctx context.Context, deploymentName string, p
 	return err
 }
 
+func (a *apiServer) workerServiceIP(ctx context.Context, deploymentName string) (string, error) {
```

Comment:
This is more of a question, but am I right in understanding that we can't simply use the service's name and expect it to be resolved into the correct IP?  Is that something that requires a k8s DNS setup?